docs/apps/crypto-trading.md

```diff
@@ -3,189 +3,212 @@ sidebar_position: 2
 title: Crypto Trading Bot
 ---
 
-### On GDN
+### Crypto App details
 
 | **Tenant** | **Fabric** | **Stream App** | **GUI** | **Source Code**|
 |----------- |----------|----|-----------|-----------|
 |  `demo@macrometa.io` | `_system` | `crypto-trading-app` | [**Crypto Trading GUI**](https://macrometacorp.github.io/tutorial-cryptotrading/) |[github](https://github.com/Macrometacorp/tutorial-cryptotrading)|
 
-### On GDN (Wavelength 5G)
-
-| **Tenant** | **Fabric** | **Stream App** | **GUI** | **Source Code**|
-|----------- |----------|----|-----------|-----------|
-|  `demo@macrometa.io` | `_system` | `crypto-trading-app` | [**Crypto Trading GUI**](https://macrometacorp.github.io/demo-crypto-trading-5g/) |
 
 :::note
 Stream Workers is currently an Enterprise only feature. 
 Please contact support@macrometa.com if you have any questions.
 :::
 
-**Stream App:**
+### Configure the Crypto App
+
+1. If you haven't already, [sign up for a free Macrometa Playground account](https://auth.paas.macrometa.io/signup). 
```

Comment:
Remove this step. It's covered in the following step for logging in. You might take a look at the tasks in the Streams or Collections sections to see how we do this.